src/Gridify/GridifyGlobalConfiguration.cs

```diff
@@ -24,6 +24,12 @@ public static class GridifyGlobalConfiguration
       /// </summary>
       public static bool CaseSensitiveMapper { get; set; }
 
+      /// <summary>
+      /// This option enables allows for an intermediate object to be null i.e in obj.PropA.Prob PropA can be null
+      /// Default is false
+      /// </summary>
+      public static bool AvoidNullReference { get; set; } = true;
```

Comment:
This is a great feature, although I'm afraid it might create confusion with the other similar option for collection null checks. ( `DisableNullChecks` ) I was thinking, should we somehow merge these two or not? What do you think?

Reply:
My second concern is
Right now you default AvoidNullReference to true. That’s a breaking change: anyone upgrading will suddenly get null‐safe semantics when they probably didn’t want them.

Reply:
Fair points, but I need a decision on the path to follow here. 

I can use the configuration `DisableNullChecks`, but people that already have it set it to true will have the behavior changed. 
Should you accept this change, or should we create a separate configuration for this feature?

Reply:
> I can use the configuration DisableNullChecks, but people that already have it set it to true will have the behavior changed.

Ah that is also a good point ... ok let's keep your version (`AvoidNullReference `) with a default false value to avoid breaking changes. So we can release it in the next minor release. Later in `v3`, we can refactor these. 

But please make sure we explain the difference well in the summaries and also documentation. 🙏